src/main.js

```diff
@@ -430,10 +430,12 @@ function handleAppWebContentsCreated(dc, contents) {
     event.preventDefault();
     if (!isTrustedURL(url)) {
       log.info(`Untrusted popup window blocked: ${url}`);
+      shell.openExternal(url);
       return;
     }
     if (isTeamUrl(url) === true) {
       log.info(`${url} is a known team, preventing to open a new window`);
+      shell.openExternal(url);
```

Comment:
Tested and looks good, only that I'm seeing belong in the log which is not meaningful
```bash
[17:47:10.227] [info] http://localhost:8065/api/v4/image?url=https://example.com/ is a known team, preventing to open a new window
```

Reply:
I'll probably have to update the log message, as it doesn't apply in this case, originally was trying to prevent only teams from displaying

Reply:
added a separate case for api, so it is properly handled